docs/code/reference.md

```diff
@@ -225,7 +225,7 @@ If you've reported your issue against the wrong app, a developer may mark it as
 If you're reporting a "Wishlist" issue, like a feature request, a developer may mark your bug as "Opinion" or "Won't Fix". Developers are often open to discussion about these kinds of issues, but please do not harass a developer for marking your report this way.
 
 ## You Can Get a Bit of Help {#you-can-get-a-bit-of-help}
-If you're not sure about anything above, you are always welcome to our development IRC channel: #elementary-dev on irc.freenode.net. We might be able to help you track down the actual project where you should report the issue, or perhaps even aid you with any English language issue you might come across. Most developers want to help you make good bug reports.
+If you're not sure about anything above, you are always welcome to our development [IRC channel](https://webchat.freenode.net?randomnick=1&channels=%23elementary-dev): #elementary-dev on irc.freenode.net We might be able to help you track down the actual project where you should report the issue, or perhaps even aid you with any English language issue you might come across. Most developers want to help you make good bug reports.
```

Comment:
I would have put the link around the `#elementary-dev` section, not the `IRC Channel`.